src/main/java/cart/application/CouponService.java

```diff
@@ -0,0 +1,43 @@
+package cart.application;
+
+import cart.dao.CouponDao;
+import cart.domain.*;
+import cart.dto.CouponResponse;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class CouponService {
+
+    public static final int COUPON_ADD_MONEY = 100000;
```

Comment:
```suggestion
    private static final int COUPON_ADD_MONEY = 100_000;
```
상수와 구분하기 위해 공백을 밑에 추가해주면 좋겠네요!

Reply:
상수화도 놓친 게 꽤 있네요.  😂 수정하겠습니다!